7Cav-Alive.Altis/scripts/missionEnd/missionEnd_air.sqf

```diff
@@ -0,0 +1,54 @@
+private _boxNumber = missionNameSpace getVariable "endMissionNumber";

```

Comment:
What calls this file? A quick peak at the `mission.sqm` doesn't show anything referencing `missionEnd_air` or any of the other files.

Reply:
missionEnd_armor/air/mech/etc calls the variable to increase it by 1, so when it reaches 3 (0-3) the mission ends

Reply:
Sure, but what calls these files?